### ISSUE TITLE
Change multiple inline style attributes before typing

### ISSUE THREAD

**Issue by jameslesliewatts** [ueberdosis/tiptap#2039]
### What’s the bug you are facing?

I'm having an issue setting multiple stored marks before typing. For example, if I change the font family and then change the color, color will change but font family will revert. It also happens if you change alignment after changing a style(although setting the alignment first and then changing a style is fine) and with a custom font size extension I made that is modeled on the color extension.

### How can we reproduce the bug on our side?

- Go to the code sandbox listed below
- With no text selected, change a style(set the color to red)
- Before typing, change a second style(set the font family to mono)
- Notice that the first change has been reverted

### Can you provide a CodeSandbox?

https://codesandbox.io/s/tiptap-issue-template-forked-jclnu?file=/src/components/Tiptap.vue

### What did you expect to happen?

I expected inline styles to stack and all be applied to the next character I typed.

### Anything to add? (optional)

Tiptap is great! Thank you for all the hard work!

[Here](https://github.com/ueberdosis/tiptap/discussions/2038) is the original discussion that prompted this issue.

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by sagarsehwag-simpplr** [comment]
i'm also facing the same issue, i have upgraded the tiptap package to latest version `2.0.0-beta.213` which consist of the fix that @philippkuehn has mentioned, but still the issue doesn't seems to be fixed.

**Comment by jameslesliewatts** [comment]
> i'm also facing the same issue, i have upgraded the tiptap package to latest version `2.0.0-beta.213` which consist of the fix that @philippkuehn has mentioned, but still the issue doesn't seems to be fixed. Am i missing something here?
> 
> @jameslesliewatts is the fix working for you?

@sagarsehwag-simpplr Yes, when they first did the fix it started working. That was over a year ago though so if you are seeing it now it's possible there was a regression. We haven't updated to the latest version (or checked this particular functionality) yet.
